test/system/005-info.bats

```diff
@@ -187,6 +187,25 @@ host.slirp4netns.executable | $expr_path
     fi
 }
 
+@test "rootless podman with symlinked $HOME" {
+    if ! is_remote; then
+        # This is only needed as rootless, but we don't have a skip_if_root
+        # And it will not hurt to run as root.
+        ln -s /home /tmp/home
+
+        old_home=$HOME
+
+        export HOME=/tmp/home
+
+        # Just need the command to run cleanly
+        run_podman info
+
+        export HOME=$old_home
```

Comment:
no need to export just call `HOME=... run_podman info` then the env is only set for one command.